### DISCUSSION TITLE
Problem in using the latest version

### DISCUSSION THREAD

**Discussion by MostaFaramin** [yalmip/YALMIP#1336] [Problems using YALMIP]
Hi Johan,

Hope you are doing great. I have been using Yalmip for a long time. I didn't have any problem using it until I installed the latest version of Yalmip. Now, when I run a code including LMIs, I face with this problem!?! I don't know how to solve it. Please give a clue to solve it.

Thank you in advance

![Capture](https://github.com/yalmip/YALMIP/assets/63315976/3af3f4ba-f904-40ae-bae8-e732c83d3f69)

**Comment by johanlofberg** [comment]
I guess you are using octave (or a very old version of MATLAB). Edit sdpsettings and replace all occurances of "" with ''

Regarding the non-symmetry detected, read the Learn more link, to see if your model is flawed,. If you really intended to have a square non-symmetric constraints, you should write it as X(:)>=0 or triu(X)>=0 to really make it evident to your self and YALMIP that you are interested in an elementwise constraint
